templates/index.html

```diff
@@ -27,7 +27,9 @@
 
             <form method="POST" action="/subscribe">
                 <div class="row" id="form">
-                    <input type="text" placeholder="Email address" name="email"/>
+                    <input required class="name-field" type="text" placeholder="First Name" name="fname"/>
+                    <input class="name-field" type="text" placeholder="Last Name" name="lname"/>
```

Comment:
oh true, I was looking at it on my huge monitor and it looked plausible. now that I'm on my laptop, nevermind.